dkist/dataset/utils.py

```diff
@@ -24,7 +26,6 @@ def get_array_repr(array):
 
 def dataset_info_str(ds_in):
     # Import here to remove circular import
-    from dkist.dataset import TiledDataset
```

Comment:
Does this not still need to be here to avoid circular imports?

Reply:
reverted